subsys/net/lib/lwm2m/lwm2m_pull_context.c

```diff
@@ -315,6 +322,8 @@ static void do_transmit_timeout_cb(struct lwm2m_message *msg)
 {
 	int ret;
 
```

Comment:
No need for an empty line between declarations, rather add one after as checkpatch requires.